### ISSUE TITLE
Chart legend circle elements fail accessibility test as all have identical id

### ISSUE THREAD

**Issue by peteharverson** [elastic/elastic-charts#970]
The `circle` elements used in chart legends all use the identical id of `dot-a`, which causes an accessibility issue, as revealed by the [axe](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd) web accessibility testing tool.

Can be seen on https://elastic.github.io/elastic-charts/?path=/story/legend--right, or for example:

![image](https://user-images.githubusercontent.com/7405507/104346247-1de2f300-54f7-11eb-94b8-ea11a106ef4b.png)
 
This will block accessibility tests being added for https://github.com/elastic/kibana/issues/51456 on any pages which use an Elastic chart where the legend is visible.

**Comment by nickofthyme** [comment]
Thanks @peteharverson we'll fix that ASAP.

**Comment by darnautov** [comment]
hey! 👋 Not sure if https://github.com/elastic/elastic-charts/pull/971 fix suffices, because it's possible to have multiple charts with the same legend on the same page. Prepending a unique chart id to the `id` attribute should work. cc @rshen91 @nickofthyme

**Comment by rshen91** [comment]
@darnautov good call - there is a new PR without any id (no longer needed) that should address this issue - #973
